cfgov/unprocessed/apps/ccdb-search/webpack-config.js

```diff
@@ -2,70 +2,72 @@
    Settings for webpack JavaScript bundling system.
    ========================================================================== */
 
-   const TerserPlugin = require( 'terser-webpack-plugin' );
+const TerserPlugin = require( 'terser-webpack-plugin' );
 
-   /* Set warnings to true to show linter-style warnings.
+/* Set warnings to true to show linter-style warnings.
       Set mangle to false and beautify to true to debug the output code. */
-   const COMMON_MINIFICATION_CONFIG = new TerserPlugin( {
-     parallel: true,
-     extractComments: false,
-     terserOptions: {
-       ie8: false,
-       ecma: 5,
-       warnings: false,
-       mangle: true,
-       output: {
-         comments: false,
-         beautify: false
-       }
-     }
-   } );
+const COMMON_MINIFICATION_CONFIG = new TerserPlugin( {
+  parallel: true,
+  extractComments: false,
+  terserOptions: {
+    ie8: false,
+    ecma: 5,
+    warnings: false,
+    mangle: true,
+    output: {
+      comments: false,
+      beautify: false
+    }
+  }
+} );
 
-   /* Commmon webpack 'module' option used in each configuration.
+/* Commmon webpack 'module' option used in each configuration.
       Runs code through Babel and uses global supported browser list. */
```

Comment:
Trivial: align the comment to its new indentation above.

Reply:
Fixed in 2e529be4eb50be9c927b4aeb046a8eee6bddb9df